z3alpha/multiprocess_evaluator.py

```diff
@@ -16,21 +16,21 @@
 
 def run_solver(
     solver_path, smt_file, timeout, id, strategy=None, tmp_dir="/tmp/", 
```

Comment:
With the current design, the argument `strategy` and `tmp_dir="/tmp/` are not needed. Z3alpha is treated as other solvers, that `solver_path` + `arguments` + `smt_file`.